.circleci/config.yml

```diff
@@ -123,19 +123,37 @@ jobs:
           name: Link NPM sodium
           command: npm link sodium
       - run:
-          name: Link NPM argon2
-          command: npm link argon2
+          name: Build dependencies
+          command: apk add --no-cache -t build-dependencies make gcc g python libtool autoconf automake
+      - run:
+          name: Access npm folder as root
+          command: cd $(npm root -g)/npm
+      - run:
+          name: Install fs-extra
+          command: npm install fs-extra
+      - run:
+          name: Edit something
+          command: sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js
```

Comment:
i told you to delete the above lines:

     - run:
          name: Install fs-extra
          command: npm install fs-extra
      - run:
          name: Edit something
          command: sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js

Reply:
done